docs/customization/customizing-emails.mdx

```diff
@@ -0,0 +1,174 @@
+---
+sidebar_position: 6
+---
+
+# Customizing Transactional Emails
+
+This guide will teach you how to customize transactional emails provided by Solidus.
+
+Solidus has built-in transactional emails that notify customers of various events associated with their order.
+For example, the following actions can trigger an email:
+
+- Completing order checkout
+- Shipping an order
+- Cancelling an order
+- Completing a batch promtion codes creation (in the admin panel)
+
+Solidus has built-in emails for all of the above scenarios and more. However, given that the default Solidus
+emails are intended to be very plain, you will likely wish to customize them for your store.
+You may also want to add new emails, such as a welcome email when a customer creates a new account, an
+abandoned cart email when a customer leaves their cart without checking out, or a notification that a
+customer's credit cart has expired if your store uses recurring subscriptions.
+
+:::info
+
+Solidus transactional emails use [Action Mailer][action-mailer], which is built into Rails. Therefore, most
```

Comment:
I haven't checked if this kind of mark-up for links works well in the new guides, but probably it's better to be consistent and use `[text](link)` everywhere. WDYT?

Reply:
If they work, I'd keep this format and try to adopt it everywhere else too. Using `[text](link)` causes a LOT of annoying text wrapping issues in the editor.

Reply:
Fine with me

Reply:
I'm used to the `[][]` format when the link is used multiple times and the `[]()` when there's only one. What if that's the convention we use?

Reply:
Hmm, yeah, however, it's easy to forget when there's an `if n > 1` condition :slightly_smiling_face: I'm ok if we use `[]()` everywhere, maybe with the definitions at the very bottom of the page.

Reply:
Yes, the other downside of this approach is that it's easy to forget references at the end of the page when they disappear from the content, or they become only 1 occurrence.